README.md

```diff
@@ -13,69 +13,75 @@
 </p>
 <br/>
 
-Metasyn is a Python package that **generates synthetic data**, and allows **sharing of the data generation model**, to facilitate collaboration and testing on sensitive data without exposing the original data.
+Metasyn is a Python package to __generate synthetic tabular data__ in a transparent, understandable, and privacy-friendly way. Our goal is to make it possible for owners of sensitive data to create test data, do open science, improve code reproducibility, encourage data reuse, and enhance accessibility of their datasets, without worrying about leaking private information.
```

Comment:
This is a nice description, however I would propose to add a short sentence on _what_ metasyn specifically allows users to do (functionality wise), as opposed to only stating its goals. Currently it reads very lovely, especially from a marketing or engaging standpoint, but to me it lacks a concrete 'metasyn allows you to achieve all these goals by doing X'.

Reply:
cool, let me try to incorporate this.